hnn_core/gui/gui.py

```diff
@@ -481,11 +486,10 @@ def data(self):
     @staticmethod
     def load_parameters(params_fname=None):
```

Comment:
Does it make sense to have a default of `None` anymore? Does it always need a valid file name?

Reply:
Ah right. Yeah, I'll get rid of that default value. Thanks!